### ISSUE TITLE
Intellisense popup in the middle of nowhere.

### ISSUE THREAD

**Issue by IanMatthewHuff** [microsoft/vscode-jupyter#2870]
Intellisense popup in the middle of nowhere. 
<img width="674" alt="Intelli" src="https://user-images.githubusercontent.com/812783/67045775-cc756980-f0e3-11e9-907a-4ecc7de02a48.PNG">
<img width="630" alt="Intelli2" src="https://user-images.githubusercontent.com/812783/67045925-15c5b900-f0e4-11e9-8b96-74acc3b9587f.PNG">
Seems like this might be related to microsoft/vscode-python#8061 or at least that is a good way to repro it.

Repro:

1. Get in edit mode (note selection mode seems ok)
2. Shift enter in a cell with long output
3. Your cursor is in the next cell, but that cell is off screen
4. Start to type (I was initially hitting the j key to move up)
5. Popups start to appear in odd spots.

Basically just get cursor into edit mode. Scroll offscreen and type.
